### ISSUE TITLE
Include license file in source distribution

### ISSUE THREAD

**Issue by synapticarbors** [wimglenn/oyaml#12]
Could the license file be included in the source distribution? See:

https://docs.python.org/2/distutils/sourcedist.html#the-manifest-in-template

**Comment by wimglenn** [comment]
Hi @synapticarbors : `License: MIT` is listed in the `PKG-INFO` [metadata](https://packaging.python.org/specifications/core-metadata/#license) for source distribution, is that enough for you? If not, I will get a new sdist out with the license file.

**Comment by synapticarbors** [comment]
@bgruening it's not part of the tarball on pypi where the recipe is being built from. We can manually package it by copying the file from the repo and putting it into the conda-forge recipe, but it's nice to just have it in the source distribution in case there are changes, we would automatically pick them up.

**Comment by bgruening** [comment]
Ah I see, I just checked the github tarball. Would be nice to have it in the pypi tarball as well, that's true.

**Comment by wimglenn** [comment]
I've tagged/released the changes in https://github.com/wimglenn/oyaml/pull/13 and put `oyaml-0.7.tar.gz` and `oyaml-0.7-py2.py3-none-any.whl` to PyPI.  There is no change in the wheel (and v0.6 wheel was already including license file) but unfortunately I had to version bump the wheel otherwise pip will get confused and collect the sdist thinking it's newer. @synapticarbors does this resolve / can this issue be closed now?
